transients_nearby_galaxies/transients_nearby_galaxies.ipynb

```diff
@@ -0,0 +1,1182 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    size=9
</pre><p> </p><p>We should justify our selection. "A 9x9 grid will include our target with enough pixels to establish a background" or something to that effect</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>